styles.css

```diff
@@ -0,0 +1,67 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: Inter, Verdana, Geneva, Tahoma, sans-serif;
+  min-width: 1400px;
+  margin: 0;
+}
+
+.nav-bar li a {
+  color: blue;
+  font-weight: 100;
+  letter-spacing: 2px;
+  text-decoration: saddlebrown;
+  padding: 20px 55px;
+  display: inline-block;
+  width: 100%;
+  text-align: center;
+  transition: all 0.5s;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: center;
+  flex-shrink: inherit;
+  list-style: none;
+}
+
+.main-content {
+  background-image: url(img/Header-llustration-desktop@2x.jpg);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding-left: 30%;
+  padding-right: 30%;
+  min-height: 90vh;
+}
+
+.heading {
+  padding-top: 170px;
+}
+
+.footer {
+  list-style: none;
+  display: flex;
+  width: 50%;
+  padding-left: 1px;
+}
+
+li:hover {
+  text-decoration: aqua;
+}
+
+.fab {
+  padding: 10px;
+}
+
+.green {
+  color: green;
+}
```

Comment:
- [ ] Great implementation on this project however, this section or branch is for setup and mobile version skeleton not for desktop or tablet. 
![port18](https://user-images.githubusercontent.com/55635977/129956712-0d0d0d1e-2b42-4213-9aed-6069655e2c74.png)

Kindly, work on the mobile section first, make sure the page doesn't scroll left or overflow-x: and display the project information in the mobile pixels. check for the image above

Reply:
You can go to icons website and download from there